debian/changelog

```diff
@@ -1,6 +1,9 @@
 opx-nas-interface (5.18.0+opx4) unstable; urgency=medium
```

Comment:
Shouldn't we bump up the version?

Reply:
Also need to bump the version in configure.ac